src/www/views/servers_view/server_list_item/server_card/index.ts

```diff
@@ -140,10 +146,9 @@ const getSharedComponents = (element: ServerListItemElement & LitElement) => {
     ServerConnectionState.RECONNECTING,
   ].includes(server.connectionState);
   const hasErrorMessage = Boolean(server.errorMessageId);
-
-  const messages = {
+  const messages: {[key: string]: string} = {
     serverName: server.name,
-    error: hasErrorMessage ? localize(server.errorMessageId) : '',
+    error: hasErrorMessage ? localize(server.errorMessageId) : server.errorMessageId,
```

Comment:
Why is this changing?

Reply:
That's legacy, and used for show errors on bad configs. We introduced that error message when we removed support for non-AEAD ciphers, and wanted to communicate that the server was no longer supported. Perhaps we can remove that.

I hear you about the unsatisfying UI. There's a lot that can be improved, but the UI changes need more careful thought and work with UXD, and it doesn't need to block this feature.

For now, we may want to consider a dedicated message for the session config errors. But that can be in a separate PR.